src/integrationTest/java/eu/xenit/gradle/testrunner/Reproductions.java

```diff
@@ -152,4 +152,14 @@ public void testIssue133Extension() throws IOException {
     public void testIssue133Label() throws IOException {
         testProjectFolder(REPRODUCTIONS.resolve("issue-133-label"), ":buildDockerImage", true);
     }
+
+    @Test
+    public void testIssue173Empty() throws IOException {
+        testProjectFolder(REPRODUCTIONS.resolve("issue-173-empty"), ":buildDockerImage");
+    }
```

Comment:
Ik snap dat hier iets met gradle gebouwd en gestest wordt door middel van de method calls te volgen maar ik kan niet terug vinden wat er precies gestest wordt. Waar staat issue-173-empty ergens in de src?

Als je kan wijzen naar waar die REPRODUCTIONS.resolve("issue-173-empty") naar wijst kan ik dit misschien achterhalen.